apps/sms/services/test/unit/moz_mobile_message/messaging_service_test.js

```diff
@@ -0,0 +1,670 @@
+/*global bridge,
+         MocksHelper
+*/
+
+'use strict';
+
+require('/services/js/bridge_service_mixin.js');
+require('/services/test/unit/mock_bridge.js');
+require('/views/shared/test/unit/mock_messages.js');
+require('/views/shared/test/unit/mock_settings.js');
+require('/views/shared/test/unit/mock_smil.js');
+
+var MocksHelperForAttachment = new MocksHelper([
+  'bridge',
+  'Settings',
+  'SMIL'
+]).init();
+
+var MessagingService, successEvt, errorEvt;
+
+suite('Messaging service >', function() {
+
+  MocksHelperForAttachment.attachTestHelpers();
+
+  suiteSetup(function(done) {
+    window.self = {};
+    window.importScripts = sinon.stub();
+
+    require('/services/js/moz_mobile_message/messaging_service.js', done);
```

Comment:
Agree, I think in my past tests for SharedWorker, I used this approach (requiring script in ```suiteSetup```), because I was adding event listeners to ```self/window``` object, but now we have ```bridge``` abstraction that does this, so we can use usual flow here.